packages/aria-voyager/tests/listbox/navigation/focus/with-single-selection.test.ts

```diff
@@ -13,7 +13,7 @@ test('focus activates the first selection item', () => {
 
   list.focus();
 
-  expect(list.getAttribute('aria-activedescendant')).toBe(secondItem.id);
-  expect(firstItem.getAttribute('aria-current')).toBeNull();
-  expect(secondItem.getAttribute('aria-current')).toBe('true');
+  await expect.element(list).toHaveAttribute('aria-activedescendant', secondItem.id);
```

Comment:
this looks great. But unfortunately does not combat the present flakiness with the tests.

Can you rewrite the `await expect.element()` with their `await expect.poll()` versions.